cmd/main.go

```diff
@@ -220,21 +220,25 @@ func (c *nodeLocalController) SetupReconcilers(mgr manager.Manager, opts *nnf.Op
 		return err
 	}
 
+	semNnfNodeStorageDone := make(chan int, 1)
+	semNnfNodeStorageDone <- 1
 	if err := (&controllers.NnfNodeStorageReconciler{
 		Client:            mgr.GetClient(),
 		Log:               ctrl.Log.WithName("controllers").WithName("NnfNodeStorage"),
 		Scheme:            mgr.GetScheme(),
 		SemaphoreForStart: semNnfNodeBlockStorageDone,
+		SemaphoreForDone:  semNnfNodeStorageDone,
 	}).SetupWithManager(mgr); err != nil {
 		return err
 	}
 
-	// The NLC controllers that do not use EC do not need startup coordination.
+	// The NLC controllers that depend on EC to be ready.
```

Comment:
```suggestion
	// The NLC controllers that depend on EC need to be ready.
```

Reply:
That suggestion changes the meaning of the comment, making it incorrect.  I'm trying to say that following this comment you'll find the controllers that don't use EC directly, but do depend on it having done its work before they can be started.